Makefile

```diff
@@ -1,7 +1,7 @@
 default: fmt vet errcheck test
 
 test:
-	go test -v -timeout 60s -race ./...
+	go test -v -timeout 2m -race ./...
```

Comment:
The consumer tests are rather fast, it is that other tests that use the older mock broker logic (the `Returns()` plus the default handler to be more specific) sometimes take longer to complete. The thing is that in the original mock broker implementation it would delay a response to a request **indefinitely** until a reply is provided via `Returns()` method. Now if a mock broker happens to get a request before `Returns()` is called it waits for `expectationTimeout` for `Returns()` to be called and then ignores the request making it IO timeout on the consumer side. Consumer retries eventually but that adds up to the execution time.

I will try to increase the `expectationTimeout` and bring the overall test suite timeout to the original value.